testcases/armnn.yaml

```diff
@@ -9,29 +10,8 @@
 
 {% block test_target %}
   {{ super() }}
-    - repository:
-        metadata:
-          format: Lava-Test Test Definition 1.0
-          name: armnn test
-          description: "armnn test"
-          os:
-          - debian
-          scope:
-          - functional
-        run:
-          steps:
-          - dhclient
-          - apt-get install -y ntp
-          - wget --no-check-certificate {{ ARMNN_TARBALL_URL }}
-          - tar xf armnn.tar.xz
-          - cd home/buildslave/workspace/armnn-ci-build
-          - export BASEDIR=`pwd`
-          - cd $BASEDIR/build
-          - ln -s $BASEDIR/build/libprotobuf.so.15.0.0 ./libprotobuf.so.15
-          - export LD_LIBRARY_PATH=`pwd`
-          - chmod a+x UnitTests
-          - lava-test-case ArmNN-Unit-Tests --shell ./UnitTests
-      from: inline
-      name: armnn
-      path: inline/armnn.yaml
+    - repository: https://github.com/Linaro/test-definitions.git
+      from: git
+      path: automated/linux/armnn/armnn-unit-tests.yaml
```

Comment:
I think you want to include
parameters:
  ARMNN_TARBALL: '{{ ARMNN_TARBALL_URL }}'

do you need to include the TEST_DIR to? or is the default ok '/usr/local/bin' ?

Reply:
ARMNN_TARBALL has been added as a param, I'll leave TEST_DIR out for now as the default should be alright.